app/src/main/java/it/niedermann/owncloud/notes/main/MainViewModel.java

```diff
@@ -350,7 +350,7 @@ public void postExpandedCategory(@Nullable String expandedCategory) {
                 lastSecondaryCategory.icon = NavigationAdapter.ICON_SUB_MULTIPLE;
             } else if (belongsToLastPrimaryCategory) {
                 if (isCategoryOpen) {
-                    item.label = currentPrimaryCategory + "/" + currentSecondaryCategory;
+                    item.label = currentSecondaryCategory;
```

Comment:
check for `null` and assign an empty string in this case

Reply:
yes I ve done it